delete.go

```diff
@@ -43,21 +61,21 @@ status of "ubuntu01" as "stopped" the following will delete resources held for
 		if err != nil {
 			return err
 		}
-		switch s {
-		case libcontainer.Stopped:
-			destroy(container)
-		case libcontainer.Created:
-			container.Signal(syscall.SIGKILL)
-			for i := 0; i < 100; i++ {
-				time.Sleep(100 * time.Millisecond)
-				if err := container.Signal(syscall.Signal(0)); err != nil {
-					destroy(container)
-					return nil
-				}
+		if context.Bool("force") {
+			if s == libcontainer.Stopped {
+				destroy(container)
+			} else {
+				killContainer(container)
+			}
+		} else {
+			switch s {
+			case libcontainer.Stopped:
+				destroy(container)
+			case libcontainer.Created:
+				return killContainer(container)
+			default:
+				return fmt.Errorf("cannot delete container that is not stopped: %s", s)
 			}
```

Comment:
@hqhq it does, the container is up and running after `runc create` and waiting for a `runc start` to exec the user process